main.c

```diff
@@ -0,0 +1,15 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+//function to print random numbers
```

Comment:
```suggestion
/* function to print random numbers */
```
Please use c-style comments